indico/modules/categories/templates/display/base.html

```diff
@@ -86,6 +91,19 @@ <h1 class="category-title {{ category_title_classes }}">
     </div>
 </div>
 <script>
+    $('#navigate-button').on('click', function(evt){
+        evt.preventDefault();
+        $('<div>').categorynavigator({
+            openInDialog: true,
+            category: "{{ category.id }}",
```

Comment:
I guess it shouldn't matter match, but the initial value for `category` in `categorynavigator` widget is a number. Here you are passing a string which is a bit inconsistent

Reply:
True. Thanks. Just changed it.